clang/test/Sema/aarch64-neon-vcmla-ranges.c

```diff
@@ -0,0 +1,202 @@
+// RUN: %clang_cc1 -triple aarch64-linux-gnu -target-feature +neon -target-feature +v8.3a -ffreestanding -fsyntax-only -verify %s
```

Comment:
Why isn't this test under this path ? clang/test/Sema/aarch64-neon-immediate-ranges

Reply:
Moved :+1: